tests/test_utils.py

```diff
@@ -0,0 +1,40 @@
+import os
+import numpy as np
+from tempfile import TemporaryDirectory
+from autowebcompat import utils
+
+
+def test_get_bugs():
+    Object = utils.get_bugs()
+    assert(isinstance(Object, list))
+
+
+def test_mkdir():
+    d = TemporaryDirectory()
+    direc_path = os.path.join(d.name + "\\test")
+    utils.mkdir(direc_path)
+    assert(os.path.isdir(direc_path))
+
+
+def test_load_image():
+    img = utils.load_image("Image")
```

Comment:
This file doesn't exist in the repository, so I'm not sure the test will pass.

Reply:
@marco-c  earlier i had the load_image function in the file and i was using a random dictionary to just check if the code was working, Because an empty file name would result in a parameter not declared error. Should i remove the test ?

Reply:
You can add a `parent_dir` argument to `load_image`, with the default being `data_resized`.
Then in the test you can create a fake image in the temporary directory, pass the temporary directory as parent_dir to load_image and then assert that the image is loaded as a 2x2 numpy array with the correct dimensions.